src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1454,13 +1764,8 @@ else if ( variant.getReference().length() == 1 ) {
      */
     private void sortFuncotationsByTranscriptForOutput( final List<GencodeFuncotation> funcotationList ) {
         //TODO: Make this sort go from "worst" -> "best" so we can just pop the last element off and save some time.
```

Comment:
Have you addressed these TODOs?

Reply:
No, not yet.  I was waiting to do these for when I start speeding it up.